custom-values.yaml

```diff
@@ -0,0 +1,9 @@
+environment:
```

Comment:
@sigalsax can we add a remark here above each param?

Reply:
I am deleting this file from our chart because this is not something we will be supplying for the user. they will supply us with their own `custom-values.yaml` as part of the install process. We have duplicated values in values.yaml so I will add that explanation there

Reply:
added card for shuli and tagged you: https://github.com/cyberark/conjur-docs/issues/1053